### ISSUE TITLE
Decorrelation stretch rendering does not work

### ISSUE THREAD

**Issue by aloboa** [EnMAP-Box/enmap-box#811]
Decorrelation stretch rendering does not change the display 
vs. a simple linear stretching, it does not perform any decorrelation.

version 3.13.1 Code [7a0aa3b3476](https://github.com/EnMAP-Box/enmap-box/commit/7a0aa3b34768b6b89dc0aad881648d4d438576c1)

**Comment by janzandr** [comment]
Confirmed, I get the following error:
```
An error has occurred while executing Python code: 

sklearn.utils._param_validation.InvalidParameterError: The 'feature_range' parameter of MinMaxScaler must be an instance of 'tuple'. Got [0, 255] instead. 
Traceback (most recent call last):
  File "C:\Users\Andreas\AppData\Roaming\QGIS\QGIS3\profiles\EnMAP-Box Develop\python\plugins\enmapboxplugin\enmapbox\coreapps\decorrelationstretchapp\decorrelationstretchdialog.py", line 182, in onLiveUpdate
    self.onApplyClicked()
  File "C:\Users\Andreas\AppData\Roaming\QGIS\QGIS3\profiles\EnMAP-Box Develop\python\plugins\enmapboxplugin\enmapbox\coreapps\decorrelationstretchapp\decorrelationstretchdialog.py", line 174, in onApplyClicked
    renderer = self.currentRenderer()
  File "C:\Users\Andreas\AppData\Roaming\Python\Python39\site-packages\typeguard\__init__.py", line 1033, in wrapper
    retval = func(*args, **kwargs)
  File "C:\Users\Andreas\AppData\Roaming\QGIS\QGIS3\profiles\EnMAP-Box Develop\python\plugins\enmapboxplugin\enmapbox\coreapps\decorrelationstretchapp\decorrelationstretchdialog.py", line 126, in currentRenderer
    scaler2.fit(percentiles)
  File "C:\Users\Andreas\AppData\Roaming\Python\Python39\site-packages\sklearn\preprocessing\_data.py", line 450, in fit
    return self.partial_fit(X, y)
  File "C:\Users\Andreas\AppData\Roaming\Python\Python39\site-packages\sklearn\base.py", line 1344, in wrapper
    estimator._validate_params()
  File "C:\Users\Andreas\AppData\Roaming\Python\Python39\site-packages\sklearn\base.py", line 666, in _validate_params
    validate_parameter_constraints(
  File "C:\Users\Andreas\AppData\Roaming\Python\Python39\site-packages\sklearn\utils\_param_validation.py", line 95, in validate_parameter_constraints
    raise InvalidParameterError(
sklearn.utils._param_validation.InvalidParameterError: The 'feature_range' parameter of MinMaxScaler must be an instance of 'tuple'. Got [0, 255] instead.
```

**Comment by janzandr** [comment]
Already fixed in v3.14-alpha.
